tests/dialers_test.go

```diff
@@ -44,6 +59,15 @@ func TestDialer(t *testing.T) {
 		t.Fatal("Must set -db_name")
 	}
 
+	if *token != "" || *tokenFile != "" {
```

Comment:
Would you mind adding a comment here explaining that if token/tokenFile are not set, the `github.com/GoogleCloudPlatform/cloudsql-proxy/proxy/dialers/mysql` library will automatically attempt to find default authentication via `proxy.InitDefault`?